docs/_data/home.yml

```diff
@@ -0,0 +1,2 @@
+- name: Home
+  link: /index.html
```

Comment:
I had to add this file since I wanted a relative URL that works for local setup and on github pages. So this was the quickest solution and really don't know if there is a proper fix for this. I did not find a way yet using the [Liquid](https://jekyllrb.com/docs/liquid/) template language.

Reply:
I think that's fine, don't know enough about Jekyll to give you good ideas here.